src/scaffoldmaker/meshtypes/meshtype_3d_lung1.py

```diff
@@ -461,15 +461,43 @@ def generateBaseMesh(cls, region, options):
                                 upperNodeIds[e3 + 1][e2][e1], upperNodeIds[e3 + 1][e2][e1 + 1], upperNodeIds[e3 + 1][e2 + 1][e1], upperNodeIds[e3 + 1][e2 + 1][e1 + 1]]
 
                             if (e3 < (uElementsCount3 - 1)) and (e2 == (uElementsCount2 - 1)) and (e1 == 0):
-                                # Distal-front wedge elements
+                                # Distal-front wedge elements / lung edge (inside)
```

Comment:
By inside and outside, do you mean left and right? Since the same code works for left and right lungs... suggest changing.